src/app/sell-item-page/sell-item-page.component.ts

```diff
@@ -0,0 +1,38 @@
+import { Component } from '@angular/core';
+import { DataService } from '../shared/services/data.service';
+import { Item } from '../shared/models/item.model';
+
+@Component({
+  selector: 'app-sell-item-page',
+  templateUrl: './sell-item-page.component.html',
+  styleUrls: ['./sell-item-page.component.css']
+})
+export class SellItemPageComponent {
+
+  itemName: string;
+  sellerID: number;
+  conditionTypeID: number;
+  categoryTypeID: number;
+  description: string;
+  price: number;
+  isItemSold: number;
+  rating: number;
+  pictureURL: string;
+
+  constructor(private dataService: DataService) { }
+
+  submit() {
+    this.dataService.addNewItem(
+      this.itemName,
```

Comment:
Like I stated in my comment in `data.service.ts`, you could instead make a new object that has all of these values and pass that in. That way, you only passing one argument instead of 9 arguments.